gulpfile.js

```diff
@@ -148,7 +148,9 @@ gulp.task('installFixtures', function() {
     }, 1 * 1000);
     shell.cd('test/fixtures');
 
-    execAsync('npm install --quiet', {cwd: '../fixtures'}).then(() => {
+    execAsync('type yarn &> /dev/null | yarn install || npm install --quiet', {
```

Comment:
Updated. Tested regular install on macOS.
Tried running:  `gulp installFixtures` but that doesn't work.

Can you test on Windows too please?